eduvpn/nm.py

```diff
@@ -240,6 +241,25 @@ def get_vpn_status(client: 'NM.Client') -> Tuple['NM.VpnConnectionState', 'NM.Vp
         return vpns[0].get_state(), vpns[0].get_state_reason()
 
 
+def set_default_gateway(enable: bool):
+    "If True, make the VPN connection the default gateway."
+    _logger.debug(f"setting default gateway: {enable}")
+    uuid = get_uuid()
+    assert uuid is not None
+    command = [
```

Comment:
this should happen by means of the nm lib directly, not the nmcli.

you can see how nmcli does it in the C code

https://github.com/NetworkManager/NetworkManager/blob/main/src/nmcli/connections.c

all nm lib calls should have a python equivalent.

Reply:
The latest version uses the NM api.